src/icon-manifest/languages.ts

```diff
@@ -194,6 +194,7 @@ export const languages: ILanguageCollection = {
   volt: { ids: 'volt', defaultExtension: 'volt' },
   vue: { ids: 'vue', defaultExtension: 'vue' },
   wolfram: { ids: 'wolfram', defaultExtension: 'wl' },
+  wurstlang: { ids: ['wurstlang', 'wurstscript'], defaultExtension: 'wurst' },
```

Comment:
Nice catch, but I would argue that actually the plugin repo needs to be updated to `wurstlang` since it was created before we had our website and new corporate identity.
`wurstlang` is now our preferred "id". So I will change things accordingly.

Reply:
Never the less `wurst` should be added for backwards compatibility.